src/pages/cadastroproduto.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="../styles/cadastroproduto.css">
+    <title>Cadastro-Produto</title>
+</head>
+<body>
+    <div id="main-container">
+        <h1>Cadastro de Produto</h1>
+        <form id="register-form">
+            <div class="full-box">
+                <label for="name">Nome do Produto</label>
+                <input type="text" name="name" id="name" placeholder="Digite o nome do produto">
+            </div>
+            <div class="half-box spacing">
+                <label for="name">Código</label>
+                <input type="text" name="name" id="name" placeholder="Digite o Código">
+            </div>
+            <div class="half-box">
```

Comment:
todos os inputs estão com o `name` e `id` iguais, pode alterar para que cada um tenha o nome relativo ao que realmente é. por exemplo, esse seria **codigo**.
E tem que alterar o `for` do label `também`